README.md

```diff
@@ -215,18 +216,24 @@ Lhm.cleanup(true, until: Time.now - 86400)
 
 First, get set up for local development:
 
-    git clone git://github.com/soundcloud/lhm.git
-    cd lhm
-
-To run the tests, follow the instructions on [spec/README](https://github.com/soundcloud/lhm/blob/master/spec/README.md).
-
-We'll check out your contribution if you:
+```
+dev clone lhm
+dev up
+```
 
-  * Provide a comprehensive suite of tests for your fork.
-  * Have a clear and documented rationale for your changes.
-  * Package these up in a pull request.
+To run the tests:
+```
+dev unit # unit tests
+dev int # integration tests
+dev test # all tests
+```
 
-We'll do our best to help you out with any contribution issues you may have.
+### dbdeployer
+The integration tests rely on a master/slave replication setup of MySQL.
+We're using [dbdeployer](https://github.com/datacharmer/dbdeployer) to set this up via `./install.sh`.
+`dbdeployer` provides scripts for operating and accessing the nodes in `$HOME/sandboxes/rsandbox_5_7_22`.
```

Comment:
I'm not the biggest fan that the test for this project will interfere with my `$HOME`. A directory with a name like `sandboxes` is fairly generic and could contain other stuff unrelated to mysql. This would mess with that organization.

Maybe it's better to have this install the mysql instances directly in this directory, or under some sort of `~/.directory`.